packages/pangea-node-sdk/src/services/redact.ts

```diff
@@ -99,6 +101,21 @@ class RedactService extends BaseService {
     Object.assign(input, options);
     return this.post("v1/redact_structured", input);
   }
+
+  /**
+   * @summary Unredact
+   * @description Decrypt or unredact fpe redactions.
+   * @operationId redact_post_v1_unredact
+   * @param {Object} request - Unredact request data
```

Comment:
This would be more correct:
```suggestion
   * @param {Redact.UnredactRequest} request - Unredact request data
```
but really we don't need to be putting the types in the TSDoc since they are already pulled from the TS signature.
```suggestion
   * @param request - Unredact request data
```